src/visualizer/raysAppFunctions/plotFrame.m

```diff
@@ -57,6 +131,60 @@ function plotNodes(app)
 
 end
 
+% function plotRays(app)
```

Comment:
If we don't need this anymore we can just delete these lines instead of commenting them

Reply:
Removed